lib/rules/custom-property-no-outside-root/index.js

```diff
@@ -19,7 +19,7 @@ const rule = function (actual) {
     }
 
     result.warn((
-      `'${ruleName}' has been deprecated and in 8.0 will be removed.`
+      `'${ruleName}' has been deprecated and in 8.0 will be removed. Use 'stylelint-suitcss instead'`
```

Comment:
Can you change all the result updates from `Use 'stylelint-suitcss instead'` to `Instead use the community 'stylelint-suitcss' plugin pack.`?